std/experimental/ndslice/internal.d

```diff
@@ -92,13 +92,13 @@ struct PtrShell(Range)
         mixin (`_shift ` ~ op ~ `= shift;`);
     }
 
-    auto opBinary(string op)(sizediff_t shift)
+    typeof(this) opBinary(string op)(sizediff_t shift)
```

Comment:
Yes unfortunately Dscanner can't expand mixins atm, s.t. is the poor man's workaround "for the  greater good".